binrz/rz-test/rz-test.c

```diff
@@ -192,7 +192,7 @@ int rz_test_main(int argc, const char **argv) {
 	int ret = 0;
 	char *cwd = NULL;
 	RzTestState state = { 0 };
-
+	RzPath *sys_path = rz_path_new();
```

Comment:
Check for null.

Reply:
The error return value. Don't know what this it in `rz_test_main` but something `>0`